viewflow/views/list.py

```diff
@@ -158,7 +170,11 @@ class AllArchiveListView(LoginRequiredMixin, generic.ListView):
     context_object_name = 'task_list'
 
     def get_template_names(self):
-        return 'viewflow/site_archive.html'
+        opts = self.flow_classes[0]._meta
```

Comment:
I agree, actually I also had some second thoughts and think it will more straightforward to pass app_label directly to the All*ListView. I'll update the code.